docs/en/mkdocs.yml

```diff
@@ -163,9 +164,6 @@ markdown_extensions:
     permalink: true
 - markdown.extensions.codehilite:
     guess_lang: false
-# Uncomment these 2 lines during development to more easily add highlights
```

Comment:
Why remove this?

Reply:
Haven't removed it on purpose. Just run `new-lang az` and not played with other lang fixes

Reply:
Would add it back, probably a bug that it removes the comment